### PR TITLE
Issue: CI: add size-limit #181

### PR DESCRIPTION
**Pull request by douglasferlini**
Pull request for issue https://github.com/kantord/LibreLingo/issues/181

### CODE REVIEW

File: workspaces/web/webpack.config.js [hunk 1]
```diff
@@ -144,13 +144,20 @@ module.exports = {
         },
         mode: process.env.NODE_ENV,
         performance: {
-            hints: false, // it doesn't matter if server.js is large
```
Comment (kantord) [inline workspaces/web/webpack.config.js:147]: Well, it really kinda doesn't matter if server.js is large, at least not to the same degree that it matters that client.js is large.

I think you meant to introduce a limit on client.js, not server.js.

Comment (douglasferlini) [reply workspaces/web/webpack.config.js:147]: I've changed the webpack config to limit only the client size.  Please check if it is what you was expecting.

Comment (kantord) [reply workspaces/web/webpack.config.js:147]: yes, it does make sense. How do the set limits compare to the actual numbers observed?

**Comment by kantord** [comment]
Thanks @douglasferlini!

@allcontributors please add @douglasferlini for code

**Pull request merged**